### PR TITLE
ATAC compatibility

### PR DESCRIPTION
**Pull request by rosaxma**
Changes: 
Modified predictions.smk and predict.py so that this part of the pipeline is compatible with ATAC inputs;
Added resources to each rule for running the pipeline on Sherlock. 
Test plans:
The tests still pass;
Ran the pipeline using tagAlign.gz (ATAC) files as input and got expected performance using crispri benchmarking.

### CODE REVIEW

**Comment by rosaxma** [comment]
[982a8c1](https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction/pull/122/commits/982a8c1b58c3c814070670027f1152e1da924276):
Changes:
MACS peak calling parameters.
Test plans:
The tests didn't pass because the peak calling parameters are different. 
I've tested the method using ATAC bam files and ATAC tagAlign files as input. Both generated expected results using CRISPR benchmarking.

**Comment by atancoder** [comment]
Re: Tests not passing 
If the output files look different because of the different MACs params, can you explain what the differences are? We can change the expected output to match the new test output if we can verify the test output is what we expect it to look like

**Comment by rosaxma** [comment]
> Re: Tests not passing
> If the output files look different because of the different MACs params, can you explain what the differences are? We can change the expected output to match the new test output if we can verify the test output is what we expect it to look like

The peak calls are different. If you have the inputs set up, you could potentially run this version with K562 DNAse as input and see if that changes the performance using crispr benchmark. The performance shouldn't be worse.

File: workflow/scripts/predict.py [hunk 1]
```diff
@@ -185,45 +191,41 @@ def main():
...
+        subset_columns = enhancers_column_names + ["normalized_atac"]
+        enhancers = enhancers_full.loc[:, subset_columns]
+    elif args.accessibility_feature == "DHS":
+        subset_columns = genes_columns_to_subset + ["DHS.RPKM.quantile.TSS1Kb"]
+        genes = genes.loc[:, subset_columns]
+        genes.columns = genes_column_names + ["normalized_dhs"]
+        subset_columns = enhancers_column_names + ["normalized_dhs"]
+        enhancers = enhancers_full.loc[:, subset_columns]
+    else:
+        print("The feature does not exist!")
```
[the remark this answers is not in the document]
Comment (rosaxma) [reply workflow/scripts/predict.py]: [631ee85](https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction/pull/122/commits/631ee852b6078219000b50abbde98d970132d5ef) addressed this

**Comment by rosaxma** [comment]
[7027039](https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction/pull/122/commits/70270390eeab0515c02465ba3679bb712a21a0c9) added test for tagAlign.gz inputs

**Review by atancoder — APPROVED** [review]

**Pull request merged**